package.json

```diff
@@ -116,8 +116,9 @@
   },
   "scripts": {
     "clean": "shx rm -rf html/ out/",
-    "vscode:prepublish": "npm run compile",
-    "build-types": "tsc -p src-common/ && shx cp -r src-common/ node_modules/neuron-ipe-types/",
```

Comment:
this is a regression because a lot of the other tasks rely on it. You might have not caught it because you already had types built out.